doc/conf.py

```diff
@@ -5,13 +5,11 @@
 # dir.
 #
 # The contents of this file are pickled, so don't put values in the namespace
-# that aren't picklable (module imports are okay, they're removed
+# that isn't pickable (module imports are okay, they're removed
```

Comment:
This change is incorrect. “values” are plural.

Reply:
Additionally "picklable" is what we mean, as in "able to use the `pickle` module to serialize"

It is a bit of a jargon term, but is used in this context often, including in the pickle module docs:

https://docs.python.org/3/library/pickle.html